cwl/src/main/scala/wdl4s/cwl/CwlFile.scala

```diff
@@ -239,6 +137,18 @@ case class CommandLineTool(
       declarations
     )
   }
+
+  def graphNodes: Set[GraphNode] = {
+
+    //need to gather up the step outputs and pass them into Call with inputs
+
+
+    val cwi = CallNode.callWithInputs(id.getOrElse("this is a made up call node name"), taskDefinition, Map.empty)
+
+    Set.empty[GraphNode] ++ cwi.inputs + cwi.call
+  }
+
+  def asCwl = Coproduct[Cwl](this)
 }
 
 object CommandLineTool {
```

Comment:
* Move this to `CommandLineTool.scala` and get rid of this file.
 * Clean up CommandLineTool.scala.  Probably should create separate files for model objects in their own package...

Reply:
created #185 so as not to further dilute this PR